packages/design-system-scripts/helpers/e2e/assertNoAxeViolations.js

```diff
@@ -41,23 +41,25 @@ function printViolations(violations) {
   violations.forEach(printViolation);
 }
 
-module.exports = async function assertNoAxeViolations(url, disabledRules = []) {
+module.exports = function assertNoAxeViolations(url, disabledRules = []) {
   if (url) {
-    await driver.get(url);
+    return driver.get(url)
```

Comment:
I re-structured this because I think this is partially the cause for some of the timeouts. I couldn't figure out where it was happening in the async/await syntax, but using promises seems to work.